operator/app/src/main/java/org/keycloak/operator/controllers/KeycloakRealmImportJob.java

```diff
@@ -129,6 +131,20 @@ private Job createImportJob() {
         buildKeycloakJobContainer(keycloakPodTemplate.getSpec().getContainers().get(0));
         keycloakPodTemplate.getSpec().getVolumes().add(buildSecretVolume());
 
+        var labels = keycloakPodTemplate.getMetadata().getLabels();
+
+        // The Job should not be selected with app=keycloak
+        labels.remove("app");
+        labels.put("app", "keycloak-realm-import");
+
+        // The Job should not connect to the cache
+        keycloakPodTemplate
+                .getSpec()
+                .getContainers()
+                .get(0)
+                .getEnv()
+                .removeIf(ev -> DEFAULT_DIST_CONFIG.keySet().contains(ev.getName()));
```

Comment:
Hmmm... but thinking about it more, I'm not sure we should be implicitly removing _all_ default config. Even though it might work now, it might be a problem in the future. Should we possibly rather just selectively remove/override the cache config (`cache=local`)?

Reply:
Ok, I will disable `health` as well as it's not used in the Job.